docs-guides/source/model-exporting.md

```diff
@@ -0,0 +1,111 @@
+```{eval-rst}
+.. index:: 
+    single: PyTorch; model exporting
+    single: model exporting
+    single: ExportedProgram
+```
+
+# Model Exporting
+
+The easiest way to generate ExportedProgram for your model is to use PyTorch's [torch.export.export](https://pytorch.org/docs/stable/export.html#torch.export.export). Exporting runs an example input tensor through your model, and captures the operations that are invoked as that input makes its way through the model's layers.
+
+```{admonition} Exporting Limitations
+
+Torch.export has limitations: graph breaks, data/shape-dependent control flow, and missing fake/meta/abtract kernels for operators. See [limitations of torch.export](https://pytorch.org/docs/stable/export.html#limitations-of-torch-export)
```

Comment:
I don't think we need to talk much about torch.export limitations in this doc. Also this information may become incorrect over time. I would just include that link and leave it a that.

Reply:
Revised as suggested